util/common/agent_util_unix.go

```diff
@@ -180,16 +180,22 @@ func StopAgent() {
 	log.Printf("Agent is stopped")
 }
 
-func ReadAgentOutput(d time.Duration) string {
+func ReadAgentLogfile(logfile string) string {
+	out, err := os.ReadFile(logfile)
+	if err != nil {
+		log.Fatal(fmt.Sprint(err) + string(out))
+	}
+	return string(out)
+}
+
+func RecreateAgentLogfile(logfile string) {
 	out, err := exec.Command("bash", "-c",
```

Comment:
nit: I know this was done before, but why are we using bash for something as simple as removing and recreating a file? Can't we just use golang for this?

Reply:
It is simpler to run a single command, but this also assumes we have bash access, so maybe doing it with golang commands is better.

Reply:
Ah right. Different file permissions.